tests/test_node_provider_db.py

```diff
@@ -139,6 +139,14 @@ def test_channel_lookup_crud():
     assert ('test-dummy-principal-2', 'dummy-slack-channel-2', 'dummy-telegram-chat-2', 'dummy-telegram-channel-2') in channels
     assert ('test-dummy-principal-1', 'dummy-slack-channel-3', 'dummy-telegram-chat-3', 'dummy-telegram-channel-3') in channels
 
+    channels = node_provider_db.get_channels_as_dict()
+    assert channels['test-dummy-principal-1'] == \
+        {'node_provider_id': 'test-dummy-principal-1', 'slack_channel_name': 'dummy-slack-channel-3', 
+         'telegram_chat_id': 'dummy-telegram-chat-3', 'telegram_channel_id': 'dummy-telegram-channel-3'}
+    assert channels['test-dummy-principal-2'] == \
+        {'node_provider_id': 'test-dummy-principal-2', 'slack_channel_name': 'dummy-slack-channel-2', 
+         'telegram_chat_id': 'dummy-telegram-chat-2', 'telegram_channel_id': 'dummy-telegram-channel-2'}
+
     # Delete channels. This deletes all channels for a given principal
     node_provider_db._delete_channel_lookup('test-dummy-principal-1')
     node_provider_db._delete_channel_lookup('test-dummy-principal-2')
```

Comment:
Probably for a separate PR, but should we make it more explicit that this command has the potential to delete more than one entry?

Reply:
Good point.
I think, long term, it might be best to have a separate table for 
- email_lookup (we already have this)
- slack_channel_lookup
- telegram_chat_lookup
- telegram_chat_lookup

Where each entry has a surrogate key, like so
```sql
CREATE TABLE IF NOT EXISTS email_lookup (
    id SERIAL PRIMARY KEY,
    node_provider_id TEXT,
    email_address TEXT
);
```